src/vos/tests/evt_ctl.c

```diff
@@ -121,7 +121,7 @@ static struct evt_desc_cbs	ts_evt_desc_nofree_cbs  = {
 };
 
 static void
-ts_open_create(void **state)
+ts_open_create()
```

Comment:
should be ts_open_create(void)

Same comment for other functions definitions.  This ensures a compiler error if user actually tries to pass a parameter.

Reply:
okay, I'm sending the patch